backend/api/Areas/Admin/Profiles/RoleProfile.cs

```diff
@@ -1,6 +1,5 @@
 using AutoMapper;
 using Pims.Api.Models;
-using Pims.Api.Profiles.Extensions;
```

Comment:
nice, I was wondering about why these were added but didn't think complaining about them added much value. Nice to see that they have been removed.